SwiftWisdomTests/StandardLibrary/Numbers/UnsignedInteger+ExtensionsTests.swift

```diff
@@ -19,4 +19,188 @@ class UnsignedIntegerExtensionTests: XCTestCase {
         let mixed = -100...100
         XCTAssert(mixed.contains(random(inRange: mixed)))
     }
+
+    // TODO: Write test
+    func testRandom() {
+        
+    }
+    
+    func test_ip_initData() {
+        var data = Data(byte: UInt8(0))
+        var number = UInt8.init(ip_data: data)
+        XCTAssertEqual(number, UInt8(0))
+        XCTAssertNotEqual(number, UInt8(1))
```

Comment:
This else case is here, and in the rest of the files, are redundant. This is because XCTAssertNotEqual is simply an alias to XCTAssertEqual with the assertion flipped from true to false. IF XCTAssertEqual is true, then number can't be anything else. I'd like some other people to weigh in on this though.